src/screens/DomainScreen/index.js

```diff
@@ -322,7 +323,7 @@ const styles = StyleSheet.create({
   list: {flex: 1},
   dummyItem: (heightParam) => ({
     height: heightParam,
-    backgroundColor: COLORS.gray9
+    backgroundColor: colors.gray1
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color value has been changed from `COLORS.gray9` to `colors.gray1`. Make sure that this change in color value is intentional and aligns with the overall theme of the application.
```diff
-    backgroundColor: COLORS.gray9
+    backgroundColor: colors.gray1
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->